aeneas/src/ir/Ir.v3

```diff
@@ -40,6 +41,7 @@ class IrClass extends IrItem {
 		}
 		return m;
 	}
+	def accept<E, R>(v: IrVisitor<E, R>, env: E) -> R { return v.visitIrClass(this, env); }
```

Comment:
I think I'd prefer not introducing visitors for this; a match on the type is locally more compact and usually keeps the logic in one place.